themes/jb/layouts/partials/episode/subscribe.html

```diff
@@ -0,0 +1 @@
+<a href="{{.CurrentSection.Params.subscribe_link}}" class="button is-info" style="margin-top:5px">Subscribe to {{.CurrentSection.Title}}</a>
```

Comment:
So, there's actually a way to pull in all the needed info from hugo functions only. That way we don't have to hard-code the subscribe page (making it a bit more portable (i.e. if we change the top level `/show/` url to something like `/shows/`, currently with this we'd have to modify all the URLs to `/shows/`)). This was it'd just automatically pickup the proper subscribe folder relatively from the current "parent" (which is the show's folder)

```suggestion
{{ with .Parent.GetPage "subscribe.md" }}
<a href="{{ .Permalink }}" class="button is-info" style="margin-top:5px">Subscribe to {{.Parent.Title}}</a>
{{ end }}
```

So, what I'm doing here is since you're passing in the [current context ](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/346/files#diff-49394a321ee8a9fb9089d8aefa3bcc80803c828f1251a37919b8b97c00914b8eR38) (which is the `.` at the end of the partial function)
1. I can get the "Parent" (or the page directly "above" the current file (i.e. if we're saying that the current page is `/show/self-hosted/79/` then the parent would be `/show/self-hosted/`).
2. Then I tell the Parent that I want to use the [GetPage function](https://gohugo.io/functions/getpage/) which gets the `subscribe.md` page, and allows us access to everything about that page (i.e. the full path to the subscribe page (`.Permalink`))
3. We still want the Show's title though, so I access the `.Parent` again to get the Show's title instead of the subscribe page's title.

With these changes, you can remove the hard coded addresses for each page. (I'll suggest it to make it easier and allowing you to accept it [from the GitHub web-UI](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)).